### PR TITLE
Cosmos adapter updates for gas and IBC

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Combine tx amounts in cw adapter when denoms match
- Update CosmosTokenAdapters with latest from warp ui
- Fix IBC-to-warp adapter for IGP support
- Add logos for cosmos related chains

### Backward compatibility

Yes

### Testing

Live in production already

### CODE REVIEW

File: typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts [hunk 1]
```diff
@@ -38,28 +50,135 @@ export class NativeTokenAdapter
...
-        revisionHeight: 0n,
-      },
-      timeoutTimestamp: 0n,
-      memo: '', // how to encode this?
+      sender: transferParams.fromAccountOwner,
+      receiver: transferParams.recipient,
+      // Represented as nano-seconds
+      timeoutTimestamp: Long.fromNumber(
+        new Date().getTime() + COSMOS_IBC_TRANSFER_TIMEOUT,
+      ).multiply(1_000_000),
```
Comment (nambrot) [inline typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:119]: No way we can just use BigNumber here?

File: typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts [hunk 2]
```diff
@@ -38,28 +50,135 @@ export class NativeTokenAdapter
...
     return {
       typeUrl: '/ibc.applications.transfer.v1.MsgTransfer',
-      // @ts-ignore
-      value: transfer,
+      value,
+    };
+  }
+}
+
+export class CosmIbcToWarpTokenAdapter
```
Comment (nambrot) [inline typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:129]: I don't quite understand the class hierachy here, is `CosmIbcTokenAdapter` used?

Comment (nambrot) [reply typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:129]: Oh for just plain IBC transfers? Some kind of comment to that effect would be helpful

File: typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts (hunk 1 above)
Comment (jmrossy) [reply typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:119]: No, I tried a bunch of different things. They want Longs

File: typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts (hunk 2 above)
Comment (jmrossy) [reply typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:129]: Correct.

Comment (jmrossy) [reply typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:135]: Added comments

File: typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
+import {
+  IHypTokenAdapter,
+  ITokenAdapter,
+  TransferParams,
+  TransferRemoteParams,
+} from './ITokenAdapter';
 
-// Interacts with IBC denom tokens
-export class NativeTokenAdapter
+const COSMOS_IBC_TRANSFER_TIMEOUT = 60_000; // 1 minute
```
Comment (nambrot) [inline typescript/sdk/src/token/adapters/CosmosTokenAdapter.ts:19]: IMO, lets have a higher timeout, IIRC the default is 10 minutes

**Review by nambrot — APPROVED** [review]

**Pull request merged**